### DISCUSSION TITLE
Importing jalali cause error

### DISCUSSION THREAD

**Discussion by nasringh** [gpbl/react-day-picker#2340] [Support]
According to this document https://daypicker.dev/docs/localization#jalali-calendar I started to create a component in my project but is cause an error and it does not work

## Code 

`Cannot find module 'react-day-picker/jalali' or its corresponding type declarations.`

```tsx
import { faIR } from "date-fns/locale";
import { DayPicker } from "react-day-picker/jalali";
```

I changed it to below:
```
import { faIR } from "date-fns/locale";
import { DayPicker } from "react-day-picker/src/jalali";
```
but it showed me this error again : `Module not found: Error: Package path ./src/jalali is not exported from package `

I think jalali does not work; what can I do ?

**Comment by gpbl** [comment]
@nasringh are you sure you installed v9? 

https://stackblitz.com/edit/react-day-picker-9-2332-kqxtmr?file=src%2FApp.tsx

How do you build your project?
